seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json

```diff
@@ -390,6 +390,117 @@
       "type": "backend",
       "challengeType": 0,
       "translations": {}
-    }  
+    },  
+    {
+      "id": "589a69f5f9fc0f352b528e6f",
+      "title": "Social Authentication with OAuth",
+      "description": [
+        [
+          "",
+          "",
```

Comment:
`challengeType` 7 tells react it is a `step` challenge and to use the `<Step /> component as the view. This is what gives the intro page a `next` button.

OK, so they are placeholders, not a problem.